src/main/java/com/mercadopago/client/point/PointClient.java

```diff
@@ -0,0 +1,92 @@
+package com.mercadopago.client.point;
+
+import static com.mercadopago.MercadoPagoConfig.getStreamHandler;
+import static com.mercadopago.serialization.Serializer.deserializeFromJson;
+
+import com.mercadopago.MercadoPagoConfig;
+import com.mercadopago.client.MercadoPagoClient;
+import com.mercadopago.core.MPRequestOptions;
+import com.mercadopago.exceptions.MPApiException;
+import com.mercadopago.exceptions.MPException;
+import com.mercadopago.net.HttpMethod;
+import com.mercadopago.net.MPHttpClient;
+import com.mercadopago.net.MPRequest;
+import com.mercadopago.net.MPResponse;
+import com.mercadopago.resources.point.PointPaymentIntent;
+import com.mercadopago.serialization.Serializer;
+import java.util.logging.Logger;
+import java.util.logging.StreamHandler;
+
+/** Client that use the Point APIs. */
+public class PointClient extends MercadoPagoClient {
+  private static final Logger LOGGER = Logger.getLogger(PointClient.class.getName());
+
+  private static final String PAYMENT_INTENT_URL =
+      "/point/integration-api/devices/%s/payment-intents";
+
+  /** Default constructor. Uses the default http client used by the SDK. */
+  public PointClient() {
+    this(MercadoPagoConfig.getHttpClient());
+  }
+
+  /**
+   * Constructor used for providing a custom http client.
+   *
+   * @param httpClient httpClient
+   */
+  public PointClient(MPHttpClient httpClient) {
+    super(httpClient);
+    StreamHandler streamHandler = getStreamHandler();
+    streamHandler.setLevel(MercadoPagoConfig.getLoggingLevel());
+    LOGGER.addHandler(streamHandler);
+    LOGGER.setLevel(MercadoPagoConfig.getLoggingLevel());
+  }
+
+  /**
+   * Method responsible for creating a payment intent.
+   *
+   * @param deviceId device id
+   * @param request attributes used to create a payment intent
+   * @return payment intent information
+   * @throws MPException an error if the request fails
+   * @see <a
+   *     href="https://www.mercadopago.com/developers/pt/reference/integrations_api_paymentintent_mlb/_point_integration-api_devices_deviceid_payment-intents/post">api
+   *     docs</a>
+   */
+  public PointPaymentIntent createPaymentIntent(String deviceId, PointPaymentIntentRequest request)
```

Comment:
Estava verificando que o deviceId é um campo obrigatório, podemos adicionar uma anotação para forçar a inclusão desse parâmetro, como @NonNull do Lombok. Não vi algo que valide isso.
```suggestion
  public PointPaymentIntent createPaymentIntent(@NonNull String deviceId, PointPaymentIntentRequest request)
```

Reply:
o que é o device id? é necessário para a API de point? se for, não sei se é o ideal Eltin, porque se um dia não for mais obrigatório na API, o SDK terá que ser modificado
caso tente-se criar sem passar o deviceId, a API deve retornar 400 e o integrador verá a resposta da API

Reply:
Também não acho que devemos incluir validações como essas no SDK. A ideia é que o SDK seja apenas um wrapper que facilite a integração, mas as validações necessárias são responsabilidades de cada API.

@gdeandradero `device id` é o id da maquininha Point.

Reply:
@gdeandradero e @rhames07 sobre o isso deixar de ser obrigatório pode ocorrer como pode não ocorrer, o que vale é o presente. O caso do deviceID é que ele é atributo utilizado no path da requisição e sem isso pode até bater na API e ela avisar que não existe um recurso, mas o integrador deverá passar por tudo isso para saber que era o obrigatório o campo.